src/inspect_ai/log/_transcript.py

```diff
@@ -184,8 +184,8 @@ class ToolEvent(BaseEvent):
     arguments: dict[str, JsonValue]
     """Arguments to function."""
 
-    internal_name: str | None = Field(default=None)
-    """Internal name for tool (if any)."""
+    internal: object | None = Field(default=None)
```

Comment:
I typically do `dict[str,Any]` for these, what's your take on whether that would be okay here?

Reply:
It's not since one call site uses `str`.

Reply:
What about using Pydantic's `JsonValue` then?

```python
JsonValue: TypeAlias = Union[
        List['JsonValue'],
        Dict[str, 'JsonValue'],
        str,
        bool,
        int,
        float,
        None,
    ]
```